### DISCUSSION TITLE
[Feature] Slideshow on mobile app

### DISCUSSION THREAD

**Discussion by AlessandroTischer** [immich-app/immich#12465] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello everyone! I've been using Immich for some months now, and I still can't figure out if it's me or the mobile app doesn't have a slideshow option. On the web it work ok, but having to close the app and open the same album from the web just because I decided to start a slideshow is not the best. Am I missing something or is it the app that does miss this feature? Thanks!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4084
